templates/patterns/page_header/page_header.ui_patterns.yml

```diff
@@ -58,6 +65,12 @@ page_header:
           label: "How the Commission is organised"
         - href: "#"
           label: "News"
+    background_image:
+      type: "array"
+      label: "Background image"
+      description: "Attributes of the image to render"
+      preview:
+        src: "https://inno-ecl.s3.amazonaws.com/media/examples/example-image8.jpg"
```

Comment:
Since the ECL twig template only accepts an URL, let's just call this `background_image_url` and only expect an url, without the need for a extra `{ src: "..." }`.

Reply:
done